roadmap/2022.md

```diff
@@ -89,3 +89,55 @@ Meilisearch releases:
   - [ ] Add pagination to the multiple API resources (keys, tasks, indexes, documents).
   - [ ] Changes related to the dumps resources
   - [ ] Add PATCH support to Java SDK.
+
+## Q3
+
+Meilisearch releases:
+- [ ] [v0.28 work](https://github.com/meilisearch/integration-guides/issues/205):
+  - [ ] New API namings, could impact multiple resources and nomenclatures, like dumps, keys, search, and indexes.
+  - [ ] New behavior to update keys
+  - [ ] HTTP verbs changes
+  - [ ] Add pagination to the multiple API resources (keys, tasks, indexes, documents).
+  - [ ] Changes related to the dumps resources
+  - [ ] Add PATCH support to Java SDK.
+
+Misc:
+- [ ] Add analytics to these integrations:
```

Comment:
The clients need to be updated to allow passing a custom header, I don't know if it's something you want to mention, because it still will be an important job

Reply:
Hum, I thought it was implied since I listed all the integrations below. I'll improve the description of the topic `Add analytics to these integrations`.

Reply:
Actually, I'm not sure if is worthy to add @curquiza since "add analytics" means, having analytics in these integrations, the way how we will do that is not something we must add here in my POV (it seems to tightly couple with the implementation, not to the objective).